docs/application/native/guides/notification/minicontrol.md

```diff
@@ -3,7 +3,7 @@
 
 Minicontrol is a small application view that can be shown on the quick panel or lock screen.
 
-This feature is supported in mobile applications only.
+This feature is supported in mobile and wearable applications only.
```

Comment:
In case of native, when mobile & wearable are all supported,
we don't need to keep this sentence.

So, please remove this line.